test/e2e/common/node/container_probe.go

```diff
@@ -534,7 +533,7 @@ var _ = SIGDescribe("Probing container", func() {
 			FailureThreshold:    1,
 		}
 
-		pod := gRPCServerPodSpec(nil, livenessProbe, "etcd")
+		pod := gRPCServerPodSpec(nil, livenessProbe, "agnhost")
```

Comment:
did you run the test? You still hasn;t updated the port in liveness probe definition on the line 527

Reply:
Oh sorry about that. I changed it to 5000